packages-backend/express/src/auth.spec.js

```diff
@@ -47,3 +47,55 @@ describe('getConfiguredAudience', () => {
     );
   });
 });
+
+describe('writeSessionContext', () => {
+  const mockIssuer = 'https://mc-api.nowhere.com';
+
+  it('should not include anything in request session if no "decoded_token" exists in the request', () => {
+    const request = {};
+
+    writeSessionContext(request);
+
+    expect(request).not.toHaveProperty('session');
+  });
+
+  it('should write "userId" and "projectKey" in request session property', () => {
```

Comment:
Nit: it look like this is basically the same as the other test. We can just have one with all possible properties.

Reply:
I'm sorry but I'm not sure I understand how to merge both tests: the first one would test that, if no `decoded_token` exists in the request, no error is raised and no `session` property is attached to the request. for the second one we want to test that the `session` property is attached to the request with the required info.
I thought having both tests would be more readable.

So what you would prefer is to have one test where we provided different tokens (all three cases currently tested) and the different expectations?

Reply:
I mean the test below, for the "userPermissions".

Reply:
Sorry for not being super clear. I pushed a commit with what I had in mind https://github.com/commercetools/merchant-center-application-kit/pull/2782/commits/12a73919ca7a7761b0a7cdbb52cdbc6116e302a4

<img width="893" alt="image" src="https://user-images.githubusercontent.com/1110551/186889016-779cc396-ba71-4734-8b2d-e4c148a4f99a.png">

Essentially all these properties can be tested individually, for the sake of the test they don't have any relation with each other.

Reply:
Ok. I wanted to test both of them together as that's what we will get at runtime, but I'm also ok with this simpler approach.

Thanks for contirbuting 🙇